client/src/views/mobile/HomeMobileView.tsx

```diff
@@ -86,6 +91,11 @@ const HomeMobileView = ({
                 value={name}
                 sx={{ mx: 'auto' }}
               />
+              <Tooltip title={fixBruteAppearance ? t('unlockBruteAppearance') : t('lockBruteAppearance')}>
+                <IconButton onClick={() => setFixBruteAppearance((prev: boolean) => !prev)} size="small" sx={{ float: 'right' }}>
```

Comment:
You should never force type a callback parameter, as it's already typed by the setter itself